cmd/zeno/zeno.go

```diff
@@ -222,7 +223,7 @@ func main() {
 
 			for i := 0; i < len(leaders); i++ {
 				client := clients[i]
-				for j := 0; j < 20; j++ { // TODO: make concurrency tunable and don't fail if there are ongoing queries
+				for j := 0; j < *clusterQueryConcurrency; j++ { // TODO: don't fail if there are ongoing queries past the allowed concurrency
```

Comment:
In fixing the unit test, I realized that this concurrency is now too low. Since query execution is now effectively serialized, we might as well make this a large number because all it's doing is accepting new queries, not actually processing them in parallel. While I was at it, I made this configurable.